src/views/CloneRepo.vue

```diff
@@ -1,20 +1,66 @@
 <template>
   <ViewWrapper title="Clone down a repository">
-    <div>
+    <label for="repo-url-input">
+      <strong>
+        Enter url:
+      </strong>
+    </label>
+    <div class="repo-url-select">
+      <input v-model="url" class="repo-url-input" placeholder="Enter url" />
+      <button @click="clone">Clone</button>
```

Comment:
Disable this button if the repo already exists locally. Display some text informing the user as to why the button is disabled. Refer to the "Create branch" tool tip at the top of the app.